### DISCUSSION TITLE
[Ideas] [2025-02-03] split 방식 및 auxiliary pairwise loss

### DISCUSSION THREAD

**Discussion by JaeHyun11** [boostcampaitech7/level4-recsys-finalproject-hackathon-recsys-05-lv3#59] [Idea]
### 아이디어 제목

논문의 실험 설명

### 아이디어 설명

> Large Language Models as Data Augmenters for Cold-Start Item Recommendation
*논문에서는 LLM으로부터 얻어낸 BPR Loss를, 기존 추천 모델의 주 학습 목표 i.e.) bpr loss, softmax loss에 auxiliary loss로 더해서 함께 최적화합니다.*

**1. Time-based split**
- 전체 데이터에서 특정 날짜(혹은 특정 시점)를 기준으로 7:3 비율로 데이터를 나눔.
Train Set: 분할 지점 이전의 상호작용(사용자-아이템) 데이터
Test Set: 분할 지점 이후의 상호작용 데이터
- Cold-start item 정의: 테스트 세트에만 등장하고 훈련 세트에는 존재하지 않는 아이템을 cold-start 아이템으로 정의

**2. User Query**
- 사용자가 과거에 상호작용한 아이템들의 제목/설명을 프롬프트로

**3. LLM을 통한 BPR Loss 계산**
- 콜드스타트 아이템 샘플링
- LLM prompt 구성
- pos/neg 쌍별 선호도 계산

**4. Auxiliary BPR Loss로 추가**
- 기존 추천 모델의 주 손실에 BPR 손실을 더해 보조 작업으로 함께 최적화
